test/molecule-role/molecule/default/create.yml

```diff
@@ -18,6 +20,10 @@
         from_port: 7077  # StackState Receiver port
         to_port: 7077
         cidr_ip: '0.0.0.0/0'
+      - proto: tcp
+        from_port: 125  # StackState Receiver port
```

Comment:
comment is misleading here, this is `directional connections test port`

Reply:
and maybe we should make an ansible variable out of it, considered that is used in multiple places

Reply:
Good catch, I'll do this